test/extensions/secondorder/diag_ggn/diag_ggn_settings.py

```diff
@@ -261,27 +259,44 @@
         "target_fn": lambda: classification_targets((4,), 5),
         "id_prefix": "nested-branching-convolution",
     },
+    {
+        "input_fn": lambda: rand((7, 10)),
+        "module_fn": lambda: Sequential(
+            Linear(10, 5),
+            ReLU(),
+            # skip connection
+            Parallel(
+                Identity(),
+                Linear(5, 5),
+            ),
+            # end of skip connection
+            Sigmoid(),
+            Linear(5, 3),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(reduction="mean"),
+        "target_fn": lambda: classification_targets((7,), 3),
+        "id_prefix": "old-branching-test",
+    },
```

Comment:
I will remove this because there is a similar setting [here](https://github.com/tjs111/backpack/blob/93d91df60a353e37547c44a18e728ebe0b15c4bc/test/extensions/secondorder/diag_ggn/diag_ggn_settings.py#L191-L206)